packages/@react-spectrum/sidenav/test/SideNav.test.js

```diff
@@ -86,13 +86,13 @@ describe('SideNav', function () {
   let scrollHeight;
 
   beforeAll(function () {
-    jest.useFakeTimers();
+    jest.useFakeTimers('modern');
     scrollHeight = jest.spyOn(window.HTMLElement.prototype, 'scrollHeight', 'get').mockImplementation(() => 48);
     stub1 = jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 200);
     stub2 = jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 400);
   });
   afterAll(function () {
-    jest.useRealTimers();
+    jest.useRealTimers('modern');
```

Comment:
I don't think we need to pass 'modern' to useRealTimers, or at least I'm not finding any documentation suggesting it.

Reply:
We do need it since this test is not mocking requestAnimationFrame and only worked because the test was running this long.

modern timers do mock requestAnimationFrame. The test is now more accurate since it uses a single (fake) clock. Previously it was mixing the fake and real clock.

Reply:
I get it for useFakeTimers, but when we switch back, I don't see anything that useRealTimers takes any argument?

Reply:
Oh now I see what we're talking about. Yeah that didn't make sense 
```suggestion
    jest.useRealTimers();
```